blivet/formats/fs.py

```diff
@@ -386,7 +386,12 @@ def _getFormatOptions(self, options=None, do_labeling=False):
         argv.append(self.device)
         return argv
 
-    def doFormat(self, options=None):
+    def _preCreate(self, **kwargs):
+        super(FS, self)._preCreate(**kwargs)
+        if not self.mkfsProg:
+            return
```

Comment:
It would be nice if all ``_pre*`` functions returned bool values. If that's already the intention, the above should return False.